examples/nextjs-moose/SETUP_GUIDE.md

```
@@ -0,0 +1,332 @@
+
+### Deploy Next.js to Vercel
+
+1. Push the monorepo to GitHub
+2. Import in Vercel, set root directory to `web/`
+3. Set the build command to compile moose first:
+
+```
+cd .. && pnpm build:moose && cd web && pnpm build
+```
+
+4. Set environment variables for the production ClickHouse connection:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)